components/fee-button.js

```diff
@@ -127,7 +127,12 @@ export default function FeeButton ({ ChildButton = SubmitButton, variant, text,
   return (
     <div className={styles.feeButton}>
       <ActionTooltip overlayText={!free && total === 1 ? '1 sat' : feeText}>
-        <ChildButton variant={variant} disabled={disabled} nonDisabledText={feeText}>{text}</ChildButton>
+        <ChildButton
+          variant={variant} disabled={disabled}
+          appendText={feeText}
+          submittingText={free || !feeText ? undefined : 'paying...'}
+        >{text}
+        </ChildButton>
```

Comment:
Slightly better UX, so they have so info that we are working on submitting the payment.